### PR TITLE
Add minimal stats to riak_pipe

### PR DESCRIPTION
**Pull request by russelldb**

### CODE REVIEW

**Comment by russelldb** [comment]
Attach to the node and run

```
riak_pipe_stat:get_stats().
```

You'll need https://github.com/basho/riak_core/tree/gh412-more-kv-stats in your deps.
If you add https://github.com/basho/riak_kv/tree/gh412-tree-stats too, you'll see pipe stats in the stats blob at `/stats` and `riak-admin status`

File: src/riak_pipe_builder_sup.erl [hunk 1]
```diff
@@ -59,8 +59,10 @@ new_pipeline(Spec, Options) ->
         {ok, Pid, Ref} ->
             case riak_pipe_builder:pipeline(Pid) of
                 {ok, #pipe{sink=#fitting{ref=Ref}}=Pipe} ->
+                    riak_pipe_stat:update({create, Pid}),
                     {ok, Pipe};
                 _ ->
+                    riak_pipe_stat:update(create_error),
                     {error, startup_failure}
             end;
         Error ->
```
Comment (beerriot) [inline src/riak_pipe_builder_sup.erl:68]: Should it also update `create_error` down here, if the call to `supervisor:start_child` fails?

File: src/riak_pipe_builder_sup.erl [hunk 2]
```diff
@@ -1,4 +1,4 @@
-%% -------------------------------------------------------------------
+% -------------------------------------------------------------------
```
Comment (beerriot) [inline src/riak_pipe_builder_sup.erl]: Looks like a rogue C-d crept into your buffer.

**Comment by beerriot** [comment]
Other than my two small comments, this looks like a great start to pipe stats. I checked it out on the console, with riak-admin status, and on /stats. :+1:

**Comment by russelldb** [comment]
Added a commit that addresses the comments, so merging due to prior :+1: (as soon as the branches this depends on a merged.)

**Pull request merged**